src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -411,11 +410,23 @@ def is_monotonic_increasing(self) -> None:
         See Also
         --------
         Index.is_monotonic_decreasing : Check if the values are equal or decreasing
+
+        Examples
+        --------
+        >>> pd.Index([1, 2, 3]).is_monotonic_increasing
+        True
+        >>> pd.Index([1, 2, 2]).is_monotonic_increasing
+        True
+        >>> pd.Index([1, 3, 2]).is_monotonic_increasing
+        False
         """
-        # TODO: SNOW-1458134 implement is_monotonic_increasing
+        return (
+            self._query_compiler.is_monotonic_increasing(index=True)
```

Comment:
I think another thing you could have done is just implement `is_monotonic_increasing/decreasing` for Series and done
`return self.to_series().is_monotonic_increasing()`

Reply:
Will change it to that!